### PR TITLE
Added UI Test for dismissing account notification

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-12755
affects: none
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1004110&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

[MBL-12755](https://instructure.atlassian.net/browse/MBL-12755)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 94.86% | 94.86% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: TestsFoundation/TestsFoundation.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -220,7 +219,6 @@
 			isa = PBXGroup;
 			children = (
 				7DAC287D22B19D5C00C0B692 /* XCTest.framework */,
-				F92C1A2922838400007E0942 /* AppFramework.framework */,
```
Comment (armstrongnate) [inline TestsFoundation/TestsFoundation.xcodeproj/project.pbxproj]: 👏

**Pull request merged**